code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -510,6 +520,7 @@
 	color = "#CF3600"
 	metabolization_rate = 0.1
 	penetrates_skin = 1
+	taste_message = "almonds"
```

Comment:
They say cyanide smells like almonds but does it _taste_ like them? I mean has anyone reported such before they most likely dropped dead?

Also can Botany grow almonds? I don't recall but they probably should